azavea/temperate
# Feature/kak/choose action before wizard#418

## Overview

Show one or two dialogues for action selection before opening the action wizard.

First, ask user to choose from recommended actions or not; if user chooses to pick from recommendations, then go to dialog with recommendations. Both have options can route directly to the wizard.


### Demo

Opened from an 'action steps' page card:
![image](https://user-images.githubusercontent.com/960264/34965881-7ee8e5e0-fa25-11e7-9734-a7780153e3fc.png)

First screen:
![image](https://user-images.githubusercontent.com/960264/34965810-f355bc42-fa24-11e7-9a09-f67a78174de2.png)

If user selects to pick from recommended options:
![image](https://user-images.githubusercontent.com/960264/34965815-02cb49b2-fa25-11e7-984e-055b02c996b7.png)

On header box hover:
![popover](https://user-images.githubusercontent.com/960264/34965855-4f179faa-fa25-11e7-8128-720502b2b77f.png)


### Notes

URL paths include a parameter for passing the risk ID. This is to present the risk information in the header on the second dialog, and also will be needed to pass to the wizard, once the wizard supports editing.

Refactored to move adaptive need box into `shared`, and added it to the header on the second dialog, as in the wireframes. The header with the risk summary and popover is also in the wizard wireframes, but has not been implemented.

Task does not include listing the action cards.

## Testing Instructions

 * Go to http://localhost:4210/actions
 * Click 'take action' for one of the risks
 * Should get first dialog
 * Clicking body of dialog should go to second dialog
 * Hover on header box on second dialog should show risk popover
 * Alternate buttons on both should route to the wizard
 * Cancel button should work

Closes #418
